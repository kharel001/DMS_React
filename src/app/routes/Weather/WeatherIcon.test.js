import React from 'react';
import WeatherIcon from './WeatherIcon';
import {
    configure,
    shallow,
    mount
} from 'enzyme';

import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}


describe('<WeatherIcon />', () => {
  let wrapper;

  let store;

  beforeEach(() => {
    
  })

  afterEach(() => {
    jest.clearAllMocks();
  });
 


it("WeatherIcon snapshot", () => {
        // let wrapper = setup(dummy_data);
    wrapper = shallow( < WeatherIcon  />)

        expect(wrapper).toMatchSnapshot();

  
})

})