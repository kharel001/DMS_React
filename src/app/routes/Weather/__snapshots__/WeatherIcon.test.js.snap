// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherIcon /> WeatherIcon snapshot 1`] = `ShallowWrapper {}`;
